src/overlays/actors/ovl_En_Fish/z_en_fish.c

```diff
@@ -853,7 +853,7 @@ void func_8091F5A4(Actor* thisx, PlayState* play) {
         func_8091D7C4(this);
         Actor_MoveWithGravity(&this->actor);
         if (this->unk_248 != 0) {
-            u32 temp = (play->sceneNum ^ SCENE_LABO) != 0;
+            u32 temp = (play->sceneId ^ SCENE_LABO) != 0;
```

Comment:
huh?

Reply:
`(play->sceneId ^ SCENE_LABO)` is 0 if `sceneId == SCENE_LABO`, non-zero otherwise. Wonder if there's a better way to generate that codegen

Reply:
Yeah `u32 temp = (play->sceneId != SCENE_LABO);` works just fine.